docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>I don't see any of the cross-references I suggested in <a href="https://github.com/pymc-devs/pymc/pull/5721#issuecomment-1146344919" target="_blank">this comment</a> (nor here nor in the other cases below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks for your feedback @OriolAbril ! I think I managed to:
- [x] Include images
- [x] Include reference within `pymc` docs.

However, I have not managed to add the cross-references. For example:
- [] How to properly link to Aesara's doc? At the moment I am pointing to the GitHub repository.
- [] How to reference to a specific external method like `aesara.dprint`? 

I have done some research but got nowhere with these two open questions. Could you please give me some hints? Thanks!

Reply:
Use the syntax I shared in https://github.com/pymc-devs/pymc/pull/5721#issuecomment-1146344919. For more context I'd recommend https://pymc-data-umbrella.xyz/en/latest/webinars/contributing_to_documentation/index.html#contributing-docs

Reply:
Got it! this was indeed very helpful! I tried to add the cross-references to all methods and functions used in the notebook. I hope I did not miss anything!